README.md

```diff
@@ -54,7 +54,12 @@
 ### Key Features <a name="key-features"></a>
 
```

Comment:
- [ ] On the transaction page I can't find any option to add transactions(I'm not sure what the categories details page is I don't think it's a transaction page, I also tried to open the transaction page with routes but it didn't work). Please make sure to add a transaction page and on the transaction page to add an option so users are able to add transactions for that category. (After fixing the issue, make sure that everything is working as per project requirements, the next reviewer will check all the other features that I'm not able to check).

![image](https://github.com/martinkarugaba/budget_app/assets/83803414/48e368d7-3ff2-4fc6-8675-3a2eb7338058)